pkg/controller/endpointslice/OWNERS

```diff
@@ -0,0 +1,13 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+- bowei
```

Comment:
would it make more sense to add the `sig-network-approvers` alias as a backstop, and the `sig-network-reviewers` alias as a backstop for reviewers?

Reply:
Good idea, thanks! Just pushed an update with those aliases added.